### PR TITLE
feat(es/parser): enable ECMAScript features in default

### PR DESCRIPTION
**Pull request by sosukesuzuki**
Enables ECMAScript features in default for parser.

Should I add some tests for this?

### CODE REVIEW

**Comment by kdy1** [comment]
No. It's fine. Thanks!

**Comment by sosukesuzuki** [comment]
Part of minifier tests failed. It seems to be related to dynamic import. But I'm not sure why it is happened...

**Comment by kdy1** [comment]
Seems like, with dynamic import enabled, the input is treated differently.
Maybe it should be added to a parser test.

**Comment by sosukesuzuki** [comment]
Thank you for telling me it. I've pushed commits that update minifier tests. How should we add about testing the parser?

**Comment by kdy1** [comment]
You can add some files to https://github.com/swc-project/swc/tree/abb14510619a97401d6fede2ebc4c28f456b97b9/ecmascript/parser/tests/span/js and update test references. It's a directory to test spans, but as it prints each ast node in a visually understandable format, so it would work.

**Comment by sosukesuzuki** [comment]
I've added span tests for dynamic import.

File: ecmascript/minifier/tests/compress/fixture/projects/wmr/archive-1/chunks/prerender.93c6f601/output.js [hunk 1]
```diff
@@ -1,70 +1,37 @@
...
         data: {
-            hello: "world",
+            hello: "world"
         },
         head: {
             title: head.title,
             lang: head.lang,
-            elements: elements,
-        },
+            elements
```
Comment (kdy1) [inline ecmascript/minifier/tests/compress/fixture/projects/wmr/archive-1/chunks/prerender.93c6f601/output.js:33]: Seems like hygiene difference. So it's fine, thanks!

**Review by kdy1 — APPROVED** [review]
LGTM. Thank you!

**Pull request merged**